scripts/deploy-beta-apm.js

```diff
@@ -68,14 +65,15 @@ module.exports = async (
   const apmBases = await deployBases([APMRegistry, Repo, ENSSubdomainRegistrar])
   log('Deployed APM bases:', apmBases)
 
-  log('Deploying DAO bases...')
-  const daoBases = await deployBases([Kernel, ACL])
-  log('Deployed DAO bases', daoBases)
-
-  log('Deploying DAOFactory without EVMScripts...')
-  const evmScriptRegistry = '0x00' // Basic APM needs no forwarding
-  const daoFactory = await DAOFactory.new(...daoBases, evmScriptRegistry)
-  log('Deployed DAOFactory:', daoFactory.address)
+  let daoFactory
+  if (daoFactoryAddress) {
```

Comment:
I guess we could also check if the `daoFactory` has EVMScripts enabled, and go back to creating one if it doesn't.

Reply:
I think not checking this is actually a feature, as someone may decide to deploy an APM instance without scripts. However, I think we should log that.